.travis.yml

```diff
@@ -150,8 +174,14 @@ script:
   - /bin/bash ./bin/utils/detect_tab_in_java_class.sh
   # run integration tests defined in maven pom.xml
   # WARN: Travis will timeout after 10 minutes of no stdout/stderr activity, which is problematic with mvn --quiet.
-  - mvn -e --no-snapshot-updates --quiet --batch-mode --show-version clean install -Dorg.slf4j.simpleLogger.defaultLogLevel=error
-  - mvn -e --no-snapshot-updates --quiet --batch-mode --show-version verify -Psamples -Dorg.slf4j.simpleLogger.defaultLogLevel=error
+  - if [ "${TRAVIS_CPU_ARCH}" == "arm64" ]; then
+      mvn clean install 2> /dev/null;
+      cd ./modules/openapi-generator-cli;
+      mvn clean install 2> /dev/null;
```

Comment:
Why are we running mvn clean install command twice?
mvn clean install command is run twice because initially we run this in root to compile whole project but jar is not made through this so we can skip this  but as we have to compile whole project it is run in root and also mvn clean install command is run in ./modules/openapi-generator-cli to created jar which is used in making docker image for openapi-generator-cli.

And why are we excluding flag -Dorg.slf4j.simpleLogger.defaultLogLevel=error?
This is so because with this flag it seems to be treating warning as error.